runtime/src/lib.rs

```diff
@@ -45,6 +45,7 @@ pub mod in_mem_accounts_index;
 pub mod inline_spl_associated_token_account;
 pub mod inline_spl_token;
 pub mod inline_spl_token_2022;
+pub mod leader_bank_notifier;
```

Comment:
btw, why `LeaderBankNotifier` is added in `solana-runtime`? i think `solana-poh` is better suited, or am i missing something?

Reply:
No you're 100% right.
My initial approach was to put this on the `Bank`, but for various reasons that didn't work as nicely. And I just never re-evaluated the module location.

Reply:
done: 66bf1e861acdb0f7e29704c83ff30cd5ec5d2041